_config/session.yml

```diff
@@ -0,0 +1,39 @@
+---
+Name: session-handlers
+---
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\Control\SessionHandler\CacheSessionHandler:
+    constructor:
+      cacheAdapter: '%$Psr\SimpleCache\CacheInterface.session-handler'
+
+  SilverStripe\Core\Cache\RedisCacheFactory:
+    constructor:
+      logger: '%$Psr\Log\LoggerInterface'
+
+  SilverStripe\Core\Cache\MemcachedCacheFactory:
+    constructor:
+      logger: '%$Psr\Log\LoggerInterface'
```

Comment:
Seems like these two should be in a separate (new?) config file, they're not specific to sessions i.e. could be used for other caches?